Source/WebCore/loader/ImageLoader.h

```diff
@@ -117,6 +118,8 @@ class ImageLoader : public CachedImageClient {
 
     VisibleInViewportState imageVisibleInViewport(const Document&) const override;
 
+    bool canReuseFromListOfAvailableImages(const CachedResourceRequest&, const String& crossOriginAttribute);
```

Comment:
Can this be marked as `static`, if so, please do it. If it cannot, please consider marking it as `const` at least.